packages/portal/portal.service.ts

```diff
@@ -19,7 +20,7 @@ export class MdcPortalService {
     private _appRef: ApplicationRef,
     private _injector: Injector) { }
 
-  createComponentRef<T>(portal: ComponentType<T>, parent: HTMLElement = document.body): ComponentRef<T> {
+  createComponentRef<T>(portal: ComponentType<T>, parent: HTMLElement | false = isBrowser() ? document.body : false): ComponentRef<T> {
```

Comment:
Is snackbar or dialog usable at all with server rendering?

Reply:
Yes, I have used snack bar as well as dialog in my project which uses Angular SSR.

Reply:
Great! Glad that's working.